Testing/TestData/neuro_inputs.xml

```diff
@@ -0,0 +1,27 @@
+<!-- 
+  This is an example of a neural network event inputs file. It is being used for unit tests
+  in the InputManagerTests.cpp file.
+
+  An event inputs file consists of an XML file with a root node `<simulator_inputs>`
+  and a `<data>` node that contains a list of events organized by `<vertex>`.
+  Other nodes can be placed at the `<data>` level for general input properties if they
+  are required by a model.
+ -->
+
+<?xml version='1.0' encoding='UTF-8'?>
+<simulator_inputs>
+  <data description="neuro_input_test" clock_tick_size="1" clock_tick_unit="usec">
```

Comment:
I hate to add something, but shouldn't this be either 0.1 msec or 100 usec, just to avoid confusion (and maybe to make a better unit test)?